PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/OriginalGraph.java

```diff
@@ -17,10 +17,14 @@ public OriginalGraph() {
   /**
    * Print a string representation of this graph.
    */
-  public void print() {
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
     for (Node node : nodes.values()) {
-      System.out.println(node);
+      sb.append(node);
+      sb.append("\n");
```

Comment:
@Pathemeous I didn't mean "appending '\n' as character" as in appending it to the string directly. I meant appending it to the StringBuilder as a character instead of appending a String (i.e. calling `Stringbuilder.append(char)` instead of `StringBuildren.append(String)`)
`sb.append(node + '\n')` would indeed be less efficient, but `sb.append(node); sb.append('\n')` is more efficient
this isn't really that important though